[0m[[0minfo[0m] [0mLoading global plugins from /Users/oliverwickham/.sbt/0.13/plugins[0m
[0m[[0minfo[0m] [0mLoading project definition from /Users/oliverwickham/Documents/git/csr/fasttrack/project[0m
[0m[[0minfo[0m] [0mSet current project to csr-fasttrack (in build file:/Users/oliverwickham/Documents/git/csr/fasttrack/)[0m

[35m--- (Running the application, auto-reloading is enabled) ---[0m

[[37minfo[0m] play - Listening for HTTP on /0:0:0:0:0:0:0:0:9000

[32m(Server started, use Ctrl+D to stop and go back to the console...)[0m

[0m[[0minfo[0m] [0mscalastyle using config /Users/oliverwickham/Documents/git/csr/fasttrack/scalastyle-config.xml[0m
[0m[[0minfo[0m] [0mProcessed 66 file(s)[0m
[0m[[0minfo[0m] [0mFound 0 errors[0m
[0m[[0minfo[0m] [0mFound 0 warnings[0m
[0m[[0minfo[0m] [0mFound 0 infos[0m
[0m[[0minfo[0m] [0mFinished in 9 ms[0m
[0m[[32msuccess[0m] [0mcreated output: /Users/oliverwickham/Documents/git/csr/fasttrack/target[0m
[0m[[0minfo[0m] [0mFormatting 1 Scala source {file:/Users/oliverwickham/Documents/git/csr/fasttrack/}csr-fasttrack(compile) ...[0m
[0m[[0minfo[0m] [0mCompiling 1 Scala source to /Users/oliverwickham/Documents/git/csr/fasttrack/target/scala-2.11/classes...[0m
[0m[[0minfo[0m] [0mscalastyle using config /Users/oliverwickham/Documents/git/csr/fasttrack/scalastyle-config.xml[0m
[0m[[0minfo[0m] [0mProcessed 66 file(s)[0m
[0m[[0minfo[0m] [0mFound 0 errors[0m
[0m[[0minfo[0m] [0mFound 0 warnings[0m
[0m[[0minfo[0m] [0mFound 0 infos[0m
[0m[[0minfo[0m] [0mFinished in 3 ms[0m
[0m[[32msuccess[0m] [0mcreated output: /Users/oliverwickham/Documents/git/csr/fasttrack/target[0m
2016-04-10 11:56:25,126 level=[INFO] logger=[application] thread=[play-internal-execution-context-1] rid=[] user=[] message=[ReactiveMongoPlugin starting...] 
2016-04-10 11:56:25,233 level=[INFO] logger=[application] thread=[play-internal-execution-context-1] rid=[] user=[] message=[Starting microservice : csr-fasttrack : in mode : Dev] 
2016-04-10 11:56:30,493 level=[WARN] logger=[application] thread=[play-internal-execution-context-1] rid=[] user=[] message=[diversity monitoring job is disabled] 
2016-04-10 11:56:30,497 level=[INFO] logger=[application] thread=[play-internal-execution-context-1] rid=[] user=[] message=[Scheduling jobs: List(SendInvitationJob after 30 seconds every 31 seconds, ExpireOnlineTestJob after 30 seconds every 31 seconds, RetrieveResultsJob after 30 seconds every 31 seconds)] 
2016-04-10 11:56:30,504 level=[INFO] logger=[play] thread=[play-internal-execution-context-1] rid=[] user=[] message=[Starting application default Akka system.] 
2016-04-10 11:56:30,505 level=[INFO] logger=[play] thread=[play-internal-execution-context-1] rid=[] user=[] message=[Application started (Dev)] 
2016-04-10 11:56:30,951 level=[WARN] logger=[connector] thread=[play-akka.actor.default-dispatcher-5] rid=[] user=[] message=[:Post:13138288:13.138ms:325734077:325.734ms:b90c:http://localhost:8100/write/audit:failed Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit] 
2016-04-10 11:56:30,951 level=[WARN] logger=[connector] thread=[play-akka.actor.default-dispatcher-5] rid=[] user=[] message=[:Post:13138288:13.138ms:325734077:325.734ms:b90c:http://localhost:8100/write/audit:failed Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit] 
2016-04-10 11:56:30,955 level=[WARN] logger=[application] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[DS_EventMissed_AuditFailureResponse : audit item : {"auditSource":"csr-fasttrack","auditType":"ResourceNotFound","eventId":"46d36b2f-6af6-4227-a782-07caa8da8277","tags":{"clientIP":"-","path":"/","X-Session-ID":"-","X-Request-ID":"-","clientPort":"-","transactionName":"Resource Endpoint Not Found"},"detail":{"method":"GET","deviceID":"-","ipAddress":"-","token":"-","referrer":"-","Authorization":"-","input":"Request to /","userAgentString":"curl/7.40.0"},"generatedAt":"2016-04-10T10:56:30.587+0000"}] exception=[java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit
	at com.ning.http.client.providers.netty.NettyConnectListener.operationComplete(NettyConnectListener.java:104) ~[async-http-client-1.8.15.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListener(DefaultChannelFuture.java:431) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListeners(DefaultChannelFuture.java:422) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.setFailure(DefaultChannelFuture.java:384) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:109) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42) ~[netty-3.9.9.Final.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_66]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_66]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_66]
Caused by: java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_66]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717) ~[na:1.8.0_66]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105) ~[netty-3.9.9.Final.jar:na]
	... 8 common frames omitted
]
java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit
	at com.ning.http.client.providers.netty.NettyConnectListener.operationComplete(NettyConnectListener.java:104) ~[async-http-client-1.8.15.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListener(DefaultChannelFuture.java:431) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListeners(DefaultChannelFuture.java:422) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.setFailure(DefaultChannelFuture.java:384) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:109) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42) ~[netty-3.9.9.Final.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_66]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_66]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_66]
Caused by: java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_66]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717) ~[na:1.8.0_66]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105) ~[netty-3.9.9.Final.jar:na]
	... 8 common frames omitted
2016-04-10 11:56:30,958 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[e4fa 2016-04-10 11:56:30.560+01:00 GET / 404 398ms] 
2016-04-10 11:56:30,978 level=[WARN] logger=[connector] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[:Post:397372962:397.372ms:1826536:1.826ms:da5c:http://localhost:8100/write/audit:failed Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit] 
2016-04-10 11:56:30,978 level=[WARN] logger=[connector] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[:Post:397372962:397.372ms:1826536:1.826ms:da5c:http://localhost:8100/write/audit:failed Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit] 
2016-04-10 11:56:30,979 level=[WARN] logger=[application] thread=[play-akka.actor.default-dispatcher-3] rid=[] user=[] message=[DS_EventMissed_AuditFailureResponse : audit item : {"auditSource":"csr-fasttrack","auditType":"RequestReceived","eventId":"a508ed29-a136-4924-b70d-8ce37dd23006","tags":{"clientIP":"-","path":"/","X-Session-ID":"-","X-Request-ID":"-","clientPort":"-","transactionName":"/"},"detail":{"statusCode":"404","method":"GET","deviceID":"-","ipAddress":"-","responseMessage":"{\"statusCode\":404,\"message\":\"URI not found\",\"requested\":\"/\"}","token":"-","referrer":"-","Authorization":"-","input":"Request to /","userAgentString":"curl/7.40.0"},"generatedAt":"2016-04-10T10:56:30.975+0000"}] exception=[java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit
	at com.ning.http.client.providers.netty.NettyConnectListener.operationComplete(NettyConnectListener.java:104) ~[async-http-client-1.8.15.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListener(DefaultChannelFuture.java:431) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListeners(DefaultChannelFuture.java:422) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.setFailure(DefaultChannelFuture.java:384) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:109) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42) ~[netty-3.9.9.Final.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_66]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_66]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_66]
Caused by: java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_66]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717) ~[na:1.8.0_66]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105) ~[netty-3.9.9.Final.jar:na]
	... 8 common frames omitted
]
java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100 to http://localhost:8100/write/audit
	at com.ning.http.client.providers.netty.NettyConnectListener.operationComplete(NettyConnectListener.java:104) ~[async-http-client-1.8.15.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListener(DefaultChannelFuture.java:431) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListeners(DefaultChannelFuture.java:422) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelFuture.setFailure(DefaultChannelFuture.java:384) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:109) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42) ~[netty-3.9.9.Final.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_66]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_66]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_66]
Caused by: java.net.ConnectException: Connection refused: localhost/127.0.0.1:8100
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_66]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717) ~[na:1.8.0_66]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152) ~[netty-3.9.9.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105) ~[netty-3.9.9.Final.jar:na]
	... 8 common frames omitted
2016-04-10 11:57:00,523 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[Executing job ExpireOnlineTestJob] 
2016-04-10 11:57:00,523 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-6] rid=[] user=[] message=[Executing job RetrieveResultsJob] 
2016-04-10 11:57:00,523 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-2] rid=[] user=[] message=[Executing job SendInvitationJob] 
2016-04-10 11:57:00,594 level=[DEBUG] logger=[application] thread=[play-akka.actor.default-dispatcher-2] rid=[] user=[] message=[Unable to take lock 'expiry-job-lock-coordinator' for '19823f63-e83b-462c-868c-d0eecb0f559f'] 
2016-04-10 11:57:00,594 level=[DEBUG] logger=[application] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[Unable to take lock 'send-invitation-job-lock-coordinator' for '19823f63-e83b-462c-868c-d0eecb0f559f'] 
2016-04-10 11:57:00,594 level=[DEBUG] logger=[application] thread=[play-akka.actor.default-dispatcher-7] rid=[] user=[] message=[Unable to take lock 'retrieve-results-job-lock-coordinator' for '19823f63-e83b-462c-868c-d0eecb0f559f'] 
2016-04-10 11:57:00,600 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-2] rid=[] user=[] message=[Completed job ExpireOnlineTestJob in 0:00:00.073: Nothing] 
2016-04-10 11:57:00,600 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-4] rid=[] user=[] message=[Completed job SendInvitationJob in 0:00:00.073: Nothing] 
2016-04-10 11:57:00,600 level=[INFO] logger=[application] thread=[play-akka.actor.default-dispatcher-7] rid=[] user=[] message=[Completed job RetrieveResultsJob in 0:00:00.073: Nothing] 
